### PR TITLE
Add git to apt-get

### PR DESCRIPTION
**Pull request by carise**

### CODE REVIEW

File: deployment-templates/py/forseti-instance.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def GenerateConfig(context):
                 'items': [{
                     'key': 'startup-script',
                     'value': """#!/bin/bash
-sudo apt-get install -y unzip
+sudo apt-get install -y unzip git
```
Comment (blueandgold) [inline deployment-templates/py/forseti-instance.py]: Should we break this up into a separate line?

Comment (mirons-google) [inline deployment-templates/py/forseti-instance.py]: I was actually going to advocate putting them all on one line.

Comment (carise) [inline deployment-templates/py/forseti-instance.py]: I can do that too

Comment (blueandgold) [inline deployment-templates/py/forseti-instance.py]: All on one line sounds good....as long as we are consistent.  :)

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request merged**